spec/features/feature_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'menu'
+require 'order'
+
+RSpec.describe 'features' do
```

Comment:
Potentially should have specific RSpec tests in here (rather than implictly testing to see if Ruby crashes) - i.e. expect statements.